control/config/fabrid.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2023 ETH Zurich
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"io"
+	"strings"
+
+	"github.com/scionproto/scion/pkg/private/serrors"
+	"github.com/scionproto/scion/pkg/segment/extensions/fabrid"
+	"github.com/scionproto/scion/private/config"
+)
+
+type FABRIDPolicy struct {
```

Comment:
I'll change this so there isn't both a LocalIdentifier and GlobalIdentifier. Also rename to FabridPolicyDefinition, so it's a bit less confusing with fabrid.Policy